src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -177,4 +211,26 @@ protected final void onShutdown() {
         // Overridden only to make final so that concrete tool implementations don't override
         super.onShutdown();
     }
+
+    /**
+     *  The emit empty loci parameter comes with several pitfalls when used incorrectly.  Here we check and either give
+     *   warnings or errors.
+     */
+    protected void validateEmitEmptyLociParameters() {
+        if (emitEmptyLoci()) {
+            if (getBestAvailableSequenceDictionary() == null) {
+                throw new UserException.MissingReference("Could not create a sequence dictionary nor find a reference.  Therefore, emitting empty loci is impossible and this tool cannot be run.  The easiest fix here is to specify a reference dictionary.");
+            }
+            if (!hasReference() && !hasIntervals()) {
+                logger.warn("****************************************");
```

Comment:
I support the warning (although I prefer not having it) because is an use case where you input the reads with header and then traverse everything that is there, including no covered regions...